test/factory/weets.js

```diff
@@ -0,0 +1,16 @@
+const { factory } = require('factory-girl');
+const faker = require('faker');
+
+const db = require('../../app/models');
+const { createUser } = require('./users');
+
+const id = () => createUser().then(user => user.dataValues.id);
+
+factory.define('Weet', db.Weet, {
+  id: factory.sequence('weet.id', n => n),
+  userId: id,
```

Comment:
This has a problem, when you do `createManyWeets()` it would create an user for every `weet`, which I don't think is the expected functionality. You can pass fixed values to the `factory.createMany()` function and do something like this
```exports.createManyWeets = (numOfWeets, attrs) => factory.createMany('Weet', numOfWeets, attrs);```
and then
``` await createManyWeets(4, {userId: /*some id**/})```

Reply:
It's better if you control to which user the weets are being added